### PR TITLE
DPROD-3901

### PR DESCRIPTION
**Pull request by Alena-deriv**
Cypress card https://app.clickup.com/t/20696747/DPROD-3901
Test scenario https://app.clickup.com/t/20696747/QATEST-1613

### CODE REVIEW

**Review by bilalmajid-deriv — CHANGES_REQUESTED** [review]
Please revert all changes relevant to the playwright installation.

**Review by bilalmajid-deriv — CHANGES_REQUESTED** [review]

File: cypress/e2e/full/signUpPage.cy.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+function enterEmail(email)
+{
+    cy.findByPlaceholderText('Email').type(email)
+    cy.findByRole('checkbox').click()
+    cy.findByRole('button', { name: 'Create demo account' }). click ()
+}
+
+function verificationIsSent () {
+    cy.findByRole('heading', { name: 'Check your email' }).should('be.visible')
+    cy.contains ('We\'ve sent a message to').should('be.visible')
```
[the remark this answers is not in the document]
Comment (Alena-deriv) [reply cypress/e2e/full/signUpPage.cy.js:25]: cy.cointains is used here as message itself contains the email address to which verification is sent and by using cy.findByText the test fails as can't locate the exact name due to different email address on each test run

[the remark this answers is not in the document]
Comment (bilalmajid-deriv) [reply cypress/e2e/full/signUpPage.cy.js:25]:
```suggestion
    cy.findByText (/We've sent a message to/).should('be.visible')
```

[the remark this answers is not in the document]
Comment (Alena-deriv) [reply cypress/e2e/full/signUpPage.cy.js:25]: Tried proposed solution but the test fails

**Review by bilalmajid-deriv — APPROVED** [review]

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**